### PR TITLE
Chat connected with send proposal and invite talent

### PR DESCRIPTION
**Pull request by lerabo**

### CODE REVIEW

File: src/components/chat/chat.tsx [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+	}, [isSuccess]);
+
+	useEffect(() => {
+		if (!isLoading) {
+			setRoomMessages(messages);
+		}
+	}, [isLoading]);
+
+	useEffect(() => {
+		const newSocket = io('http://localhost:3000');
```
Comment (lyudad) [inline src/components/chat/chat.tsx]: nonon move to envs

File: src/components/chat/chat.tsx [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+					<button type="submit">
+						<IoMdSend />
+					</button>
+				</InputBlock>
+			</ChatWrapper>
+		</Wrapper>
+	);
+};
+
+export default Chat;
```
Comment (lyudad) [inline src/components/chat/chat.tsx:201]: this component tooo large, split it

File: src/components/chat/chat.tsx (hunk 1 above)
Comment (lerabo) [reply src/components/chat/chat.tsx]: moved

File: src/components/chat/chat.tsx (hunk 2 above)
Comment (lerabo) [reply src/components/chat/chat.tsx:201]: reduced as much as I could

**Pull request merged**